sdk/spring/spring-integration-azure-core/src/main/java/com/azure/spring/integration/core/handler/DefaultMessageHandler.java

```diff
@@ -48,6 +48,8 @@ public class DefaultMessageHandler extends AbstractMessageProducingHandler {
     private final String destination;
     private final SendOperation sendOperation;
     private boolean sync = false;
+
+    @SuppressWarnings("deprecation")
     private ListenableFutureCallback<Void> sendCallback;
```

Comment:
We will refer to the `ListenableFutureCallback` usage from here https://github.com/spring-projects/spring-framework/blob/v6.0.0-RC2/spring-webmvc/src/main/java/org/springframework/web/servlet/mvc/method/annotation/DeferredResultMethodReturnValueHandler.java#L79-L92